administrator/components/com_associations/models/fields/itemlanguage.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_associations
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('JPATH_BASE') or die;
+
+JFormHelper::loadFieldClass('list');
+
+/**
+ * Form Field class for the Joomla Framework.
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class JFormFieldItemLanguage extends JFormFieldList
+{
+	/**
+	 * The form field type.
+	 *
+	 * @var        string
+	 * @since   __DEPLOY_VERSION__
+	 */
+	protected $type = 'ItemLanguage';
+
+	/**
+	 * Method to get the field options.
+	 *
+	 * @return  array  The field option objects.
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	protected function getOptions()
+	{
+		$options = array();
+
+		$input = JFactory::getApplication()->input;
+
+		$referenceId         = $input->get('id', '');
+		$associatedComponent = $input->get('acomponent', '');
+		$associatedView      = $input->get('aview', '');
+		$forcedLanguage      = $input->get('forcedlanguage', '');
+
+		if ($associatedComponent == 'com_categories')
```

Comment:
Edited it a little but as it is I don't know how you found it working :stuck_out_tongue_closed_eyes: